src/utils/mtev_confstr.c

```diff
@@ -122,13 +123,13 @@ int
 mtev_confstr_parse_boolean(const char *input, mtev_boolean *output) {
   if(!strcasecmp(input, "yes") || !strcasecmp(input, "true") || !strcasecmp(input, "on")) {
```

Comment:
Reconnoiter and thus libmtev use on/true and off/ false. If add 1/0, I
agree it should be a separate and sweeping commit.
On Apr 19, 2016 10:18 AM, "Aidan Cully" <notifications@github.com> wrote:

> In src/utils/mtev_confstr.c
> <https://github.com/circonus-labs/libmtev/pull/22#discussion_r60239196>:
>
> > @@ -122,13 +123,13 @@ int
> >  mtev_confstr_parse_boolean(const char *input, mtev_boolean *output) {
> >    if(!strcasecmp(input, "yes") || !strcasecmp(input, "true") || !strcasecmp(input, "on")) {
>
> Maybe? but I'd put that in a different PR. For what it's worth, those
> strings were mostly copied from reconnoiter
> <https://github.com/circonus-labs/reconnoiter/blob/master/src/modules/httptrap.c#L111>
> .
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/circonus-labs/libmtev/pull/22/files/50839aaa4d301ef60c21f6cb84b2ae2d99a42293#r60239196>
>